<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="titleWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>825</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>248</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="titleLable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Volnic Serial-Com Assistant For Linux</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>247</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameComSet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>598</width>
      <height>221</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>598</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>14</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPort">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>56</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>BaudRate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comBaudRate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnOpenPort">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>100</width>
       <height>68</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>OpenPort</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkTim">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator:unchecked
{	
	image: url(:/res/images/checkbox_parcial.png);
}
QCheckBox::indicator:checked
{
	image: url(:/res/images/checkbox_checked.png);
}
QCheckBox::indicator:unchecked:disabled
{
	image: url(:/res/images/checkbox_parcial_disable.png);
}
QCheckBox::indicator:checked:disabled
{
	image: url(:/res/images/checkbox_checked_disable.png);
}</string>
     </property>
     <property name="text">
      <string>Timer:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkHexSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>134</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator:unchecked
{	
	image: url(:/res/images/checkbox_parcial.png);
}
QCheckBox::indicator:checked
{
	image: url(:/res/images/checkbox_checked.png);
}
QCheckBox::indicator:unchecked:disabled
{
	image: url(:/res/images/checkbox_parcial_disable.png);
}
QCheckBox::indicator:checked:disabled
{
	image: url(:/res/images/checkbox_checked_disable.png);
}
</string>
     </property>
     <property name="text">
      <string>HEX Send</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkHexShow">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>110</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator:unchecked
{	
	image: url(:/res/images/checkbox_parcial.png);
}
QCheckBox::indicator:checked
{
	image: url(:/res/images/checkbox_checked.png);
}
QCheckBox::indicator:unchecked:disabled
{
	image: url(:/res/images/checkbox_parcial_disable.png);
}
QCheckBox::indicator:checked:disabled
{
	image: url(:/res/images/checkbox_checked_disable.png);
}</string>
     </property>
     <property name="text">
      <string>HexShow</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMsgData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>491</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>137</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearShow">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editTimInterval">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>105</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>192</y>
       <width>57</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labStatus">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>190</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #575757;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>192</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>S:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>192</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearLen">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="labSendLen">
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>190</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #575757;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
    </widget>
    <widget class="QLabel" name="labRecLen">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>190</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #575757;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenFile">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>OpenFile</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editFileName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SendFile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>StopSend</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>70</y>
      <width>201</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/images/volinci.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>598</width>
      <height>229</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
